### DISCUSSION TITLE
Thorium 2024 easter egg image contest

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium#768] [General]
Upload some images to use as an easter egg!

**Comment by trytip** [comment]
**sometimes i feel like giving up, but then i remember i have a lot of people i need to piss off**
![truedat001](https://github.com/user-attachments/assets/df7eb7a3-d38f-4c0d-a3a4-e6cd80592793)

**Reply by Alex313031** [reply]
Not memes lol

**Comment by JohnnyBob1970** [comment]
![Uploading Chromium_logo_PNG5.png…]()

**Reply by Alex313031** [reply]
Cant see it.

**Comment by JohnnyBob1970** [comment]
https://ibb.co/qss2cqZ

I will split the colors, I thought I liked the way it looked but now I see it's not a good representation 

Oh I and I have not vectorized it in Adobe Illustrator yet either

**Reply by Alex313031** [reply]
This is nice! Vector format is good, however, the final image used in the browser source must be .png

Kinda similar to the main application icon. I have a thorium.svg that I use as the source for everything. I then import it into GIMP @ 16384px x 16384px, and then downscale it to whatever size I needed, and export as .png.

I found that importing the .svg at 16384px and then downscaling produces more accurate pictures.
For example:

Importing thorium.svg at 512px, and then exporting to .png without resizing = more dithering

 &nbsp; &nbsp; &nbsp;vs.

Importing thorium.svg at 16384px, then exporting to .png after resizing to 512px = better image!

Both leave you with a 512px Thorium icon, but the second method leaves better, although slightly larger, images.

So, if you can vectorize it, great. Please send as a .svg or similar, not a .psd.
If not, a .png of the highest resolution you can create will work too.

**Comment by JohnnyBob1970** [comment]
Okay I can do that, this was just a rough draft really, there is a separation line that has to be dealt with which I will do, and I have the knowledge to do all the stuff you are asking I was a web dev and 3d animation artist for about 17 years before I switched to tattoo artist, and Im a drummer too.... But I still work as a freelancer for 3rd party subcontractors and build 3d assets for game company's from time to time, like UBISOFT NAUGHTYDOG and a couple others whenever I can get the work...
Everything slowed way down for me after Covid so I have a lot of time to burn man, I get a lil stir crazy and bored

Johnny Moore

 

    On Saturday, August 17, 2024 at 08:41:34 AM CDT, Alex F. ***@***.***> wrote:  
 
 


This is nice! Vector format is good, however, the final image used in the browser source must be .png

Kinda similar to the main application icon. I have a thorium.svg that I use as the source for everything. I then import it into GIMP @ 16384px x 16384px, and then downscale it to whatever size I needed, and export as .png.

I found that importing the .svg at 16384px and then downscaling produces more accurate pictures.
For example:

Importing thorium.svg at 512px, and then exporting to .png without resizing = more dithering

     vs.

Importing thorium.svg at 16384px, then exporting to .png after resizing to 512px = better image!

Both leave you with a 512px Thorium icon, but the second method leaves better, although slightly larger, images.

So, if you can vectorize it, great. Please send as a .svg or similar, not a .psd.
If not, a .png of the highest resolution you can create will work too.

**Comment by JohnnyBob1970** [comment]
So I just went ahead and vectorized as High Fidelity photo and exported as an svg...
But when I opened it in a browser I noticed way to many imperfections... So Im gonna fix all that
Johnny Moore

 

    On Saturday, August 17, 2024 at 07:35:44 PM CDT, Johnny Moore ***@***.***> wrote:  
 
 Okay I can do that, this was just a rough draft really, there is a separation line that has to be dealt with which I will do, and I have the knowledge to do all the stuff you are asking I was a web dev and 3d animation artist for about 17 years before I switched to tattoo artist, and Im a drummer too.... But I still work as a freelancer for 3rd party subcontractors and build 3d assets for game company's from time to time, like UBISOFT NAUGHTYDOG and a couple others whenever I can get the work...
Everything slowed way down for me after Covid so I have a lot of time to burn man, I get a lil stir crazy and bored

Johnny Moore

 

    On Saturday, August 17, 2024 at 08:41:34 AM CDT, Alex F. ***@***.***> wrote:  
 
 


This is nice! Vector format is good, however, the final image used in the browser source must be .png

Kinda similar to the main application icon. I have a thorium.svg that I use as the source for everything. I then import it into GIMP @ 16384px x 16384px, and then downscale it to whatever size I needed, and export as .png.

I found that importing the .svg at 16384px and then downscaling produces more accurate pictures.
For example:

Importing thorium.svg at 512px, and then exporting to .png without resizing = more dithering

     vs.

Importing thorium.svg at 16384px, then exporting to .png after resizing to 512px = better image!

Both leave you with a 512px Thorium icon, but the second method leaves better, although slightly larger, images.

So, if you can vectorize it, great. Please send as a .svg or similar, not a .psd.
If not, a .png of the highest resolution you can create will work too.

**Comment by JohnnyBob1970** [comment]
So I went back and reread your instructions on how you was working with your svg and exporting it to PNG final... I totally got the 16384 pixels wrong. No wonder it looks so shitty cuz I exported it at 512 X 5:12

Yahoo Mail: Search, Organize, Conquer 
 
  On Sat, Aug 17, 2024 at 10:47 PM, Johnny ***@***.***> wrote:   
So I just went ahead and vectorized as High Fidelity photo and exported as an svg...
But when I opened it in a browser I noticed way to many imperfections... So Im gonna fix all that
Johnny Moore

 

    On Saturday, August 17, 2024 at 07:35:44 PM CDT, Johnny Moore ***@***.***> wrote:  
 
 Okay I can do that, this was just a rough draft really, there is a separation line that has to be dealt with which I will do, and I have the knowledge to do all the stuff you are asking I was a web dev and 3d animation artist for about 17 years before I switched to tattoo artist, and Im a drummer too.... But I still work as a freelancer for 3rd party subcontractors and build 3d assets for game company's from time to time, like UBISOFT NAUGHTYDOG and a couple others whenever I can get the work...
Everything slowed way down for me after Covid so I have a lot of time to burn man, I get a lil stir crazy and bored

Johnny Moore

 

    On Saturday, August 17, 2024 at 08:41:34 AM CDT, Alex F. ***@***.***> wrote:  
 
 


This is nice! Vector format is good, however, the final image used in the browser source must be .png

Kinda similar to the main application icon. I have a thorium.svg that I use as the source for everything. I then import it into GIMP @ 16384px x 16384px, and then downscale it to whatever size I needed, and export as .png.

I found that importing the .svg at 16384px and then downscaling produces more accurate pictures.
For example:

Importing thorium.svg at 512px, and then exporting to .png without resizing = more dithering

     vs.

Importing thorium.svg at 16384px, then exporting to .png after resizing to 512px = better image!

Both leave you with a 512px Thorium icon, but the second method leaves better, although slightly larger, images.

So, if you can vectorize it, great. Please send as a .svg or similar, not a .psd.
If not, a .png of the highest resolution you can create will work too.

**Reply by MadAzrael84** [reply]
Sorry about posting here (I couldn't find any other way to contact) but I've been trying to download Thorium Browser for Android (the Public ARM32 one) but it keeps failing, saying "needs authorization". I'm signed in, have tried to download on 3 different browsers, and 2 different devices. So I'm like 95% certain that the problem is not on my end. I've been trying for like 2-3 days now.

**Reply by JohnnyBob1970** [reply]
> Sorry about posting here (I couldn't find any other way to contact) but I've been trying to download Thorium Browser for Android (the Public ARM32 one) but it keeps failing, saying "needs authorization". I'm signed in, have tried to download on 3 different browsers, and 2 different devices. So I'm like 95% certain that the problem is not on my end. I've been trying for like 2-3 days now.

Dont know if I should be replying to this, but I just discovered that the Dev & I are both Broken Arrow OK natives... maybe he will give me a pass on this one

Is your android set to allow install from all sources?

**Reply by MadAzrael84** [reply]
Yeah. I've tried it with allowed and not allowed both. Ended up with the same result. That's why I posted in here. I'm at a loss. LoL

**Reply by MadAzrael84** [reply]
Finally got it to download, on the wrong device. But I just bluetoothed it to the correct one. Now it won't let me log into my Google cause it says it "already exists on device".

**Reply by JohnnyBob1970** [reply]
you have Chrome installed?

**Reply by MadAzrael84** [reply]
Yeah. I planned on disabling it once I got Thorium set up. Trying to degoogle my game phone as much as possible.

**Reply by MadAzrael84** [reply]
I disabled chrome. But that didn't help cause it says I'm signed in on my device, so I dunno. Hard to tell with all the google crap on the device.

**Reply by Alex313031** [reply]
@JohnnyBob1970 Oh wow! Cool you're broken arrow.

**Reply by Alex313031** [reply]
@JohnnyBob1970 I dont know why I explained all that, you don't really need to know all that lol.

If you just send it as a .svg I can do all the stuff I need to make it in gimp.

**Reply by JohnnyBob1970** [reply]
> @JohnnyBob1970 Oh wow! Cool you're broken arrow.

I lived in Broken Arrow most of my life, use to live out on 205th AVE across from Eagle Point "considered Wagoner Co but had a Broken Arrow ph... " , Then moved into town Close to the (newer) Express way (had a pikepass) & Walmart, Indian Springs area

My wife passed away young, was married to a lil Japanese woman Lola Moore (Jones) so after Tia my daughter graduated High school in 2018 I decided to leave BA and come to MN.. You know... where it gets colder than outer space?  

My friends all want me to come back, mostly to give them tattoos haha... But I have been fortunate enough to find love again and will be moving to Saarbrücken Germany around November... 

To keep her from having to come to the US all the time... Not to sure if she likes my Drum-set though.. Says shes okay with it,.... gonna find out pretty quick Im guessin' :D.... 

So from 1 Broken Arrow Native to another... Hey wassup Brah! ;)

**Comment by rafa-ulloa** [comment]
![thorium_easter_egg](https://github.com/user-attachments/assets/94c7d5d9-8e9c-4e4e-9551-7686422a4842)
![new_thorium](https://github.com/user-attachments/assets/0f1e0af8-d5f0-49ba-a000-6b83dcde2262)

**Comment by JohnnyBob1970** [comment]
1 svg & 1 png @ 10,000+ pixels vectorized before exported to each extension... Lemme know if there is anything I can do for ya brother ;)

Johnny Moore

[](https://ibb.co/yVkPDmQ) .png only
![Chromium_logo_EXTRA_LARGE_revised](https://github.com/user-attachments/assets/6296d10c-cce7-4bbf-94b7-24834aabc62f)
![Chromium_logo_EXTRA_LARGE_revised](https://github.com/user-attachments/assets/169c7b0e-983f-4002-9e58-9d6e1a49c38e)

I like the atomic structure Rafael... that is badass idea Yo dude awesome job man. Your idea gave me another idea which is really your idea

Odd sounding but true (oxymoronic maybe)

But .... Rafael.....look at the atomic structure of "Thorium"
![thorium-atom-1802359](https://github.com/user-attachments/assets/36eb982a-c89f-4f28-8d6f-7c0a8cd0fd1f)

Isn't that badass? :D Like your idea is badassery dude, you make it come to life in an Icon or do you want me to show you my idea that is kinda more or less your idea?

Im thinking that the size of the icon is important otherwise everything else just kinda washes it out and takes your eye off the prize... IDK maybe that's just me


    On Saturday, August 24, 2024 at 04:04:48 PM CDT, Rafael Ulloa ***@***.***> wrote:  
 
 


thorium_easter_egg.png (view on web)
new_thorium.png (view on web)

**Reply by rafa-ulloa** [reply]
I like your idea. Makes a great wallpaper. I'm glad you liked the idea of the atomic structure. Thorium is an element after all.

**Comment by Alex313031** [comment]
@rafa-ulloa @Templayer @JohnnyBob1970 @trytip I will probably add all the non-winners (even satirical entries), to the thorium.rocks/contest page after this is all done, just to show off the ideas and funny shit people came up with.

Still deciding on whether I let the community vote (and how and when I would do that?), or be selfish and just pick my favourite lol.

I wish I could include them all, but that would add some megabytes to Thorium (the size of the images + an index file for each image that is stored in the .pak files in the installation dir), and it is already larger than other Chromium based browsers due to the compiler optimizations and because of the fact I include extra stuff like chromedriver and thorium_shell in the default install.

Also @Templayer that could be a cute mascot lol. But would it replace the current one on the chrome://version page, probably not lol.

**Comment by JohnnyBob1970** [comment]
hey can you give me a few more days Alex?... I have another that I am working on, but Tattoo clients currently are taking precedence over my web design skills at the moment
Johnny Moore

 

    On Friday, August 30, 2024 at 04:28:34 AM PDT, Alex F. ***@***.***> wrote:  
 
 


@rafa-ulloa @Templayer @JohnnyBob1970 @trytip I will probably add all the non-winners (even satirical entries), to the thorium.rocks/contest page after this is all done, just to show off the ideas and funny shit people came up with.

Still deciding on whether I let the community vote (and how and when I would do that?), or be selfish and just pick my favourite lol.

I wish I could include them all, but that would add some megabytes to Thorium (the size of the images + an index file for each image that is stored in the .pak files in the installation dir), and it is already larger than other Chromium based browsers due to the compiler optimizations and because of the fact I include extra stuff like chromedriver and thorium_shell in the default install.

**Reply by Alex313031** [reply]
@JohnnyBob1970 Don't worry. The winner won't be chosen for another month at least.

**Reply by JohnnyBob1970** [reply]
I still have not had time to finish mine yet

**Comment by swanknight** [comment]
An AI-generated Robo-Chrome. 😅

![Robo Chrome](https://github.com/user-attachments/assets/5087637e-8f48-4ce1-b58c-99ad0e110b02)

**Reply by JohnnyBob1970** [reply]
I really like the spherical part of this which is one that I have been workin' on kinda spherical , more toriodal  but I'm not gonna have the time to finish it up, just hit the busy time of year for me

**Comment by JackDSRS** [comment]
> Upload some images to use as an easter egg!

Actually, I made these as a custom icons for Nova Launcher. It's 512x512. No insult, but the default icon is, well, kind of basic and rough. The raws are 1920x1920.

Both use the periodic table numbering and abbreviation for thorium. One is the basic heavy grey used to reflect that side of the chemical table I think.

The second is roughly the same overlaid on the circular portion of the original Thorium icon. It's dark to retain the 3d effect and bright lettering. Were this for a PC or Mac desktop where transparency wasn't supported, I'd probably use a light effect color or just a light color since this extends to the edges of the icon edge. Maybe pull it back a bit to give it a border.

Feel free to use them, but please don't distribute them without citation. I'm sick of Alarmy selling my work. I can convert them into any format if you can't. Thanks!

![thorium 04](https://github.com/user-attachments/assets/4a1ff47c-753f-4e8e-aed4-3bdf554eaa4b)

![thorium 01](https://github.com/user-attachments/assets/92e9a033-1153-4d36-9597-3547ab0af8ef)

I don't have a single default icon on my launcher. I make my own based on photos and hand-made photorealistic items. Although the one for SD Maid is kinda PG 18.  :)

**Reply by JackDSRS** [reply]
Oh, and the dark grey one is also overlaid on the original Thorium icon as well. Although it's hard to see, there are trace colors of the blue hinted at in the grey. Like most of you, these details are lost to the eye on a desktop, but I hate cutting corners. I leave specific imperfections in now and record them as it was one of the things I cited in a case with Alarmy.

**Reply by JackDSRS** [reply]
I have to add to that grey color. I didn't want to edit my post again. The base color was picked from a sample of Thorium.

[Scary stuff. An example of the element Thorium](https://periodictable.com/Items/090.2/index.html)

[Thorium shavings. An example of the element Thorium](https://periodictable.com/Items/090.x1/index.html)

**Comment by shreyrav** [comment]
![Thorium Icon](https://github.com/user-attachments/assets/3ae79a1b-1245-4354-b20e-7ba178ef2e7c)
This is what I use as my icon for Thorium
